tools/benchmark/cmd/root.go

```diff
@@ -63,4 +70,5 @@ func init() {
 	RootCmd.PersistentFlags().StringVar(&tls.CAFile, "cacert", "", "verify certificates of HTTPS-enabled servers using this CA bundle")
 
 	RootCmd.PersistentFlags().StringVar(&user, "user", "", "specify username and password in username:password format")
+	RootCmd.PersistentFlags().DurationVar(&dialTimeout, "dial-timeout", defaultDialTimeout, "dial timeout for client connections")
```

Comment:
default to `0` so it keeps the current retry-forever behavior? 2 seconds seems kind of arbitrary